core/shared/src/main/scala/scalaz/zio/IO.scala

```diff
@@ -267,7 +278,14 @@ sealed abstract class IO[+E, +A] extends Serializable { self =>
    * it will depend on the `IO`s returned by the given continuations.
    */
   final def redeem[E2, B](err: E => IO[E2, B], succ: A => IO[E2, B]): IO[E2, B] =
-    redeem0((cause: Cause[E]) => cause.checkedOrRefail.fold(err, IO.fail0), succ)
+    redeem0((cause: Cause[E]) => cause.checkedOrRefail.fold(err, IO.fail), succ)
+
+  /**
+   * Alias for redeem
+   */
+  @deprecated("Use redeem", "scalaz-zio 0.6.0")
```

Comment:
Is this even a good idea to introduce aliases to immediately deprecate them?

Reply:
that's very intentional
people might be looking for methods with these names because they're familiar with them and they will find them both in Doc or in IDE autocompletion
but there should be one way of doing things -- that's why they're deprecated -- because there exists another, canonical, method with a name that better captures the meaning in context of ZIO